src/app.js

```diff
@@ -1 +1,80 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+console.log('Hallo daar!');
+
+
+// Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint. Log de response in de console en bestudeer de data goed: hoe is het opgebouwd?
+
+// koppel index list
+const list = document.getElementById("list-of-country`s")
+
+
+async function fetchCountry() {
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all');
+        console.log(result.data[0].name);
+
+        //sort it all
+        result.data.sort ((a,b) => (a.population - b.population));
+
+
+        //map door data
+        result.data.map((country) => {
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        result.data.map(({ flag, region, name, population }) => {
```